pyproject.toml

```diff
@@ -46,6 +47,7 @@ types-pytz = "^2021.1.0"
 types-click = "^7.1.2"
 types-PyYAML = "^5.4.3"
 types-requests = "^2.25.0"
+s3fs="^2022.10.0"
```

Comment:
We only need s3fs in the `main` section. Not in dev as well. Duplicate installs can mess things up with poetry.